back/engines/commercial/analysis/app/controllers/analysis/web_api/v1/analyses_controller.rb

```diff
@@ -79,7 +79,7 @@ def side_fx_service
         def detect_custom_fields
           container = @analysis.phase || @analysis.project
           participation_method = Factory.instance.participation_method_for(container)
-          custom_form = container.custom_form || participation_method.create_default_form!
+          custom_form = participation_method.custom_form
```

Comment:
Changed to use `participation_method.custom_form` as this always returns a fallback `new CustomForm` if the form is not persisted and then the `IdeaCustomFieldsService` will return the default fields if the form has no fields